build.gradle

```diff
@@ -210,33 +210,25 @@ tasks.withType(Jar) {
 
 test {
     outputs.upToDateWhen { false }  //tests will never be "up to date" so you can always rerun them
-    String CI = "$System.env.CI"
-    String CLOUD = "$System.env.CLOUD"
-    String SPARK = "$System.env.SPARK"
-    useTestNG{
-        if (CLOUD =="mandatory") {
+    String TEST_VERBOSITY = "$System.env.TEST_VERBOSITY"
+    String TEST_TYPE = "$System.env.TEST_TYPE"
+    useTestNG {
+        if (TEST_TYPE == "cloud") {
             // run only the cloud tests
             includeGroups 'cloud', 'bucket'
-        } else if (CLOUD == "todo") {
-            // run only the in-development cloud tests
-            includeGroups 'cloud_todo', 'bucket_todo'
-        } else if (CLOUD == "together") {
-            // run both local tests and mandatory cloud tests, together.
-            // This is good when e.g. you are done for the day and want to run tests on your machine overnight.
-            excludeGroups 'cloud_todo', 'bucket_todo'
+        } else if (TEST_TYPE == "integration"){
+            include "**/*IntegrationTest*"
```

Comment:
Think you might want to remove the last * so that it's strictly a class name suffix?